validator/src/main.rs

```diff
@@ -308,8 +309,20 @@ fn download_ledger(
     Ok(())
 }
 
+const IDENTITY_KEYPAIR_NAME: &str = "identity";
+const STORAGE_KEYPAIR_NAME: &str = "storage-keypair";
+const VOTING_KEYPAIR_NAME: &str = "voting-keypair";
```

Comment:
Drop `-keypair` suffix for storage/voting for consistent?  Or add `-keypair` to identity